recipes/future-fstrings/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "future_fstrings" %}
+{% set version = "0.4.3" %}
+{% set hash = "008a17352751afbe846ffae90313972aac099346dfcb3361466cdab7d63e12a4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - tokenize-rt
+  run:
+    - python
+    - tokenize-rt
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: https://github.com/asottile/future-fstrings
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary:
+    Adds fstrings for python<3.6
+
+  description: |
+    A backport of fstrings to python<3.6.
+
+extra:
+  recipe-maintainers:
+    - sterlingbaldwin
+    - asottile
```

Comment:
@asottile  are you okay with being a maintainer of this package on conda-forge?

Reply:
(forgive my ignorance): what does that mean for me?

Reply:
Basically we will make a new repository `conda-forge/future-strings-feedstock` that will be used to maintain this package.  Presently we have bots that will actually make pull requests when the pypi version of this package changes and rebuild for the new version.  These pull requests have to be merged by a maintainer or a conda-forge admin.  

For pure python packages the overhead is pretty.  Realistically the only thing that i can think of that could cause an issue for a package of this nature is when py37 rolls out and for some reason it no longer works.

Reply:
Seems fine, I can press green buttons :)